apps/minifront/src/state/helpers.ts

```diff
@@ -155,18 +155,7 @@ const broadcast = async (
   throw new Error('did not broadcast transaction');
 };
 
-const getTxHash = <T extends Required<PartialMessage<TransactionId>> | PartialMessage<Transaction>>(
-  t: T,
-): T extends Required<PartialMessage<TransactionId>> ? string : Promise<string> =>
-  'inner' in t && t.inner instanceof Uint8Array
-    ? (uint8ArrayToHex(t.inner) as T extends Required<PartialMessage<TransactionId>>
-        ? string
-        : never)
-    : (getTxId(t as PartialMessage<Transaction>).then(({ inner }) =>
-        uint8ArrayToHex(inner),
-      ) as T extends Required<PartialMessage<TransactionId>> ? never : Promise<string>);
-
-const getTxId = (tx: Transaction | PartialMessage<Transaction>) =>
+const txSha256 = (tx: Transaction | PartialMessage<Transaction>) =>
   sha256Hash(tx instanceof Transaction ? tx.toBinary() : new Transaction(tx).toBinary()).then(
     inner => new TransactionId({ inner }),
   );
```

Comment:
this caused problems at some point during this work, and the types were technically incorrect, so it's now simplified